src/base/solvers/solveCobraMIQP.m

```diff
@@ -278,6 +280,87 @@
            stat = -1; % Solution not optimal or solver problem
         end
         %%
+   case 'ibm_cplex'
+        % Free academic licenses for the IBM CPLEX solver can be obtained from
+        % https://www.ibm.com/developerworks/community/blogs/jfp/entry/CPLEX_Is_Free_For_Students?lang=en
+
+        cplexlp = Cplex();
+        if (~isempty(csense))
+            b_L(csense == 'E') = b(csense == 'E');
+            b_U(csense == 'E') = b(csense == 'E');
+            b_L(csense == 'G') = b(csense == 'G');
+            b_U(csense == 'G') = inf;
+            b_L(csense == 'L') = -inf;
+            b_U(csense == 'L') = b(csense == 'L');
+        elseif isfield(MILPproblem, 'b_L') && isfield(MILPproblem, 'b_U')
+            b_L = MILPproblem.b_L;
+            b_U = MILPproblem.b_U;
+        else
+            b_L = b;
+            b_U = b;
+        end
+        intVars = (vartype == 'B') | (vartype == 'I');
+        % intVars
+        % pause;
+        cplexlp.Model.A = A;
```

Comment:
Is this code from `solveCobraLP`? Why not reuse it?

Reply:
Its essentially code from solveCobraMILP.
But yes, makes sense to make a "buildCplexLP" function. I'll have a look at that.

Reply:
Yes, that would be nice - that will make the code even more maintainable.